packages/react-menu/src/common/isConformant.ts

```diff
@@ -0,0 +1,10 @@
+import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
+
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
+  const defaultOptions = {
+    disabledTests: ['has-docblock'],
+    componentPath: module!.parent!.filename.replace('.test', ''),
+  };
+
+  baseIsConformant(defaultOptions, testInfo);
+}
```

Comment:
I think that having `isConformant.ts` in packages was considered as an anti-pattern, do we really need it?

Reply:
Let's keep consistent for now and remove isConformant in a consistent way too